lib/game/tictactoe.ex

```diff
@@ -1,27 +1,13 @@
 defmodule TicTacToe do
-  @default_message "Please choose space.\n"
-
-  def start(players, board, io \\ :stdio, screen) do
-    ask_for_move(players, board, io, screen)
-  end
-
-  defp ask_for_move(players, board, io, screen, message \\ @default_message) do
+  def play_game(board, players) do
     [current_player, _] = players
 
     unless Board.is_full?(board) do
```

Comment:
Porque no `Board.full?` en lugar de `Board.is_full?`, esto es de acuerdo a una guía de estilos de Elixir?

Reply:
Si, de acuerdo a la guia:
"The names of predicate macros (compile-time generated functions that return a boolean value) that can be used within guards should be prefixed with is_." Solo le quito el "?".